client/api/src/main/java/io/smallrye/graphql/client/dynamic/api/DynamicGraphQLClient.java

```diff
@@ -14,12 +14,18 @@
 
     Response executeSync(Request request) throws ExecutionException, InterruptedException;
 
+    Response executeSync(String request) throws ExecutionException, InterruptedException;
```

Comment:
Should we not here also have a way to pass in vars and operationName ? (overload the method as they are optional). Same with the other methods.